src/main/rx/expresslrs_common.c

```diff
@@ -262,6 +272,14 @@ uint16_t txPowerIndexToValue(const uint8_t index)
     }
 }
 
+uint32_t elrsUidToSeed(const uint8_t UID[])
+{
+    return (((long)UID[2] << 24)
+        + ((long)UID[3] << 16)
+        + ((long)UID[4] << 8)
+        + UID[5]) ^ ELRS_OTA_VERSION_ID;
+}
```

Comment:
Just speculating
```suggestion
uint32_t elrsUidToSeed(const uint8_t UID[])
{
    return (uint32_t) (((unsigned long)UID[2] << 24)
        + ((unsigned long)UID[3] << 16)
        + ((unsigned long)UID[4] << 8)
        + UID[5]) ^ ELRS_OTA_VERSION_ID;
}
```

OR maybe even

```suggestion
uint32_t elrsUidToSeed(const uint8_t UID[])
{
    return (uint32_t) ((UID[2] << 24)
        + (UID[3] << 16)
        + (UID[4] << 8)
        + UID[5]) ^ ELRS_OTA_VERSION_ID;
}
```